src/zif_abapgit_definitions.intf.abap

```diff
@@ -548,4 +548,10 @@ INTERFACE zif_abapgit_definitions
       translation_languages TYPE ty_languages,
     END OF ty_i18n_params .
   TYPES ty_trrngtrkor_tt TYPE RANGE OF trkorr.
+
+  CONSTANTS:
+    BEGIN OF c_id,
+      labels TYPE string VALUE 'labels',
```

Comment:
keep `ids` on pages

Reply:
moved `labels` id to pages. Now we have some duplication within the `choose_labels` methods which I wanted to get rid. Therefore I moved the id to the global interface.

Reply:
@mbtools moved id to pages